src/parser_m3u8.rs

```diff
@@ -0,0 +1,16 @@
+use std::fs;
+
+// create a function that takes a file path and opens it
+pub fn open_file() {
+    let file_path = "./assets/tears_of_steel.m3u8";
+    let file = fs::read_to_string(file_path).expect("Unable to open file");
+
+    let lines: Vec<&str> = file.split('\n').collect();
+
+    // print each line
+    let mut line_num = 0;
+    for line in lines {
+        println!("{line_num}: {line}");
+        line_num += 1;
+    }
```

Comment:
```suggestion
    };
```
not explicitly needed but good practice